src/App/Pages/TabsPage.cs

```diff
@@ -94,6 +94,7 @@ protected override async void OnAppearing()
                 if (message.Command == "syncCompleted")
                 {
                     Device.BeginInvokeOnMainThread(async () => await UpdateVaultButtonTitleAsync());
+                    await ForcePasswordResetIfNeededAsync();
```

Comment:
⚠️ Keep in mind that given that you're in an action in the broadcaster subscription, this act as an `async void` so if `ForcePasswordResetIfNeededAsync()` throws an exception it'll crash the application. I don't know if that's what you expect, but I'd think that we should wrap this into a try...catch and log the exception and then show an alert to the user that an error happened and maybe log the user out, IDK which would be the appropriate flow if this method throws.

Reply:
added a exception log if that ever occurred.